task-force-docs/base/land.sql

```diff
@@ -75,9 +75,15 @@ SELECT
 
     -- Relevant OSM tags for land type
     MAP_FILTER(tags, (k,v) -> k IN (
+            'building',
```

Comment:
That is the expectation! But in case that some bug is introduced in the future, and we see any of those tags showing up in the source_tags collection, that would be a signal that we should investigate the issue.